src/pages/LoginToAdminPage.js

```diff
@@ -10,17 +10,50 @@ function LoginToAdminPage() {
     const [ callLoading , setCallLoading ] = useState(false);
     const [ showPasswordErrorMessage, setShowPasswordErrorMessage ] = useState(false)
     const [ playersList, setPlayersList ] = useState([]);
+    const [ password, setPassword ] = useState('');
+
+
+    // create selected players list
+    const setPlayerSelected = (index, isSelected) => {
+        // create a temporary playerlist to make sure not to changie the original.
+        const tempArray = playersList.map(player => {
+            return {
+                ...player,
+            }
+        });
+        tempArray[index].selected = isSelected;
+        // set the only seleced item's isSelected as 'true'
+        setPlayersList(tempArray);
+    }
+
+     // select all
+    const setPlayerSelectedAll = (isSelected) => {
+        const tempArray = playersList.map(player => {
+            return {
+                ...player,
+            }
+        });
+        tempArray.map((player) => ( player.selected = isSelected));
```

Comment:
`map` is when to make the array something new, not for "iterating". 
```
const newArray = tempArray.map((player) => ( player.selected = isSelected));
```
So, if it doesn't need to return a new array, `forEach` is the right choice.